widgets/src/loading_spinner.rs

```diff
@@ -6,68 +6,49 @@ live_design!{
     use link::widgets::View;
 
     pub LoadingSpinner = <View> {
-        width: Fill, height: Fill
+        width: Fill,
+        height: Fill
         show_bg: true,
         draw_bg: {
-            color: #FFFFFF00
-            instance rotation_speed: 1.0
-            instance inner_radius: 0.6 // r
-            instance outer_radius: 0.7 // R
-
+            color: #00000000
+            
+            uniform rotation_speed: 1.2
+            uniform stroke_width: 6.0
+            uniform radius: 30.0
+            uniform max_gap_ratio: 0.92
+            uniform min_gap_ratio: 0.12
+            uniform spinner_color: vec3(0.29, 0.56, 0.89)
```

Comment:
this should just be a regular color constant, which itself will resolve to a `vec4`. 

I specifically said in the last PR that we don't want to force the alpha channel to be 1.0, but you're still doing that here.

```suggestion
            uniform spinner_color: #4A8FE3
```